.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+name: Publish chart upon release creation
+
+on:
+  release:
+    types:
+      - "created"
+
+jobs:
+  release-chart:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+            fetch-depth: 0
+            submodules: recursive
+
+      - uses: cachix/install-nix-action@v22
+      - name: Pre-populate nix-shell
+        run: |
+          export NIX_PATH=nixpkgs=$(jq '.nixpkgs.url' nix/sources.json -r)
+          echo "NIX_PATH=$NIX_PATH" >> $GITHUB_ENV
+          nix-shell --pure --run "echo" ./shell.nix
+
+      - name: Remove prerelease label from dependencies
+        run: |
+            yq -i '(.dependencies[] | select(.name == "localpv-provisioner") | .version) |= sub("-prerelease$", "")' charts/Chart.yaml
```

Comment:
you might want to script this as IIRC yq messes with the blank lines or something like that

Reply:
sure.